resources/views/components-preview.blade.php

```diff
@@ -38,7 +38,7 @@ function color(variable, fallback) {
                             muted: color('--border-muted', tailwind.colors.slate[100]),
                         },
                         background: {
-                            emphasis: color('--background-emphasis', tailwind.colors.slate[200]),
+                            emphasis: color('--background-emphasis', tailwind.colors.slate[800]),
```

Comment:
I don't think this is the correct way. We have `[50]` `[100]` so the gap between `[100]` and `[800]` is to big. We have discussed this in the past with @jordythevulder and @royduin. Maybe you could add a new color variable if that is needed?

In the [core](https://github.com/rapidez/core/blob/master/resources/views/components/slideover/index.blade.php) we use `bg-black/40` why can't you use that?